include/opt-sched/Scheduler/aco.h

```diff
@@ -29,22 +31,23 @@ class ACOScheduler : public ConstrainedScheduler {
   virtual ~ACOScheduler();
   FUNC_RESULT FindSchedule(InstSchedule *schedule, SchedRegion *region);
   inline void UpdtRdyLst_(InstCount cycleNum, int slotNum);
-  // Set the initial schedule for ACO 
+  // Set the initial schedule for ACO
   // Default is NULL if none are set.
   void setInitialSched(InstSchedule *Sched);
-  
+
 private:
   pheremone_t &Pheremone(SchedInstruction *from, SchedInstruction *to);
   pheremone_t &Pheremone(InstCount from, InstCount to);
   double Score(SchedInstruction *from, Choice choice);
 
   void PrintPheremone();
 
-  SchedInstruction *SelectInstruction(std::vector<Choice> ready,
-                                      SchedInstruction *lastInst);
+  SchedInstruction *
+  SelectInstruction(const llvm::SmallVectorImpl<Choice> &ready,
+                    SchedInstruction *lastInst);
   void UpdatePheremone(InstSchedule *schedule);
-  InstSchedule *FindOneSchedule();
```

Comment:
Why is FindOneSchedule() called during the definition?

Reply:
I don't understand the question, in this LOC FindOneSchedule() is being declared, not called.